app/ts/background/settings.ts

```diff
@@ -182,26 +182,27 @@ export const setUseTabsInsteadOfPopup = async(useTabsInsteadOfPopup: boolean) =>
 export const getMetamaskCompatibilityMode = async() => (await browserStorageLocalGet('metamaskCompatibilityMode'))?.['metamaskCompatibilityMode'] ?? false
 export const setMetamaskCompatibilityMode = async(metamaskCompatibilityMode: boolean) => await browserStorageLocalSet({ metamaskCompatibilityMode })
 
-
-export async function exportSettingsAndAddressBook() {
-	const results = {
+export async function exportSettingsAndAddressBook(): Promise<ExportedSettings> {
+	const exportDate = (new Date).toISOString().split('T')[0]
+	if (exportDate === undefined) throw new Error('Datestring did not contain Date')
```

Comment:
if the intent is to compress storage size, this may also work tho the output will differ slightly (with slashes, `11/11/2023`) and should still be a valid Date string. can also get away without checks

```suggestion
	const exportDate = (new Date).toLocaleDateString('en-US')
```

Reply:
The point is to just have the date, not datetime. And I would rather use ISO date than US dates as they have weirdly month and day reversed :(